templates/custom/vesta/vesta.html.twig

```diff
@@ -116,6 +116,8 @@
     </div>
   </div>
 
+  {{ content.field_opening_hours }}
```

Comment:
No check for empty necessary?

Reply:
This is just a standard drupal field that is rendered, so it won't be rendered if it's empty 😉